config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml

```diff
@@ -3,12 +3,14 @@ kind: OpenStackDataPlaneService
 metadata:
   name: nova
 spec:
-  secrets:
-    - nova-cell1-compute-config
-    # NOTE: this Secret needs to be created before deploying the data plane.
-    # It should contain an ssh key-pair in the secret fields: ssh-privatekey
-    # and ssh-publickey
-    - nova-migration-ssh-key
+  dataSources:
```

Comment:
this should wait for the adoption job to CI such sort of changes perhaps (as this will break adoption)

Reply:
im not sure it would
we do need to modify the adoption procedure in the future to not create any custom server and just use the standard nova one

in any case i guess we can try and get the adoption procedure updated this week as well